### ISSUE TITLE
/taxifare bugs when specifying mile below a certain threshold.

### ISSUE THREAD

**Issue by essjaybee** [InZidiuZ/op-framework-bug-tracker#110]
**Reporter:**
OMGPointless#1823

**Summary:**
Been experimenting with what taxi means. Right now it seems you don't get a basic salary, so the other day I was clocked in as another job and took out my taxi. The taxi meter didn't show, but I started fiddling with the /taxi commands. Found a situation where the meter went haywire, first I dismissed this as me being clocked in as the wrong job. But today I made "Daniel" my character from another server, and made him a taxi driver. I started messing with the /taxifare parameters and I found out that if you put "mile" to less than 3, weird shit starts happening. 

**Steps to Reproduce:**
`/taxifare mile 2.0`
or
`/taxifare mile 1.0`
and drive forwards or reverse with your taxi a little so it registers mileage. (apparently also does it at 3 sometimes. might have missed this before or there are more variables to it)

**Expected Results:**
Taxi meter not to go haywire:

**Actual Results:**
It's as if the result is a division by zero, overflow or whatever and you get a really long number, alternatively a negative number.

## Additional Information

**Screenshots:**
![20190628144435_1](https://user-images.githubusercontent.com/2750716/60343076-71442d80-99b3-11e9-9095-cd6e7d6c799a.jpg)
![20190628144110_1](https://user-images.githubusercontent.com/2750716/60343127-920c8300-99b3-11e9-9d96-2bdbc261e1cf.jpg)
![20190628144043_1](https://user-images.githubusercontent.com/2750716/60343160-a6e91680-99b3-11e9-9397-04e870656f50.jpg)
![20190628144905_1](https://user-images.githubusercontent.com/2750716/60343271-e4e63a80-99b3-11e9-8f7a-4bf20cefe215.jpg)
![20190628145122_1](https://user-images.githubusercontent.com/2750716/60343516-72298f00-99b4-11e9-8458-b5b68a385548.jpg)

**Player Data:**
![20190628144928_1](https://user-images.githubusercontent.com/2750716/60343295-f16a9300-99b3-11e9-8704-ddf3dfc35abb.jpg)

**Notes:**
Nah think I got it all covered. ;)

**Comment by essjaybee** [comment]
New F8
![20190628145326_1](https://user-images.githubusercontent.com/2750716/60343543-82416e80-99b4-11e9-8726-39614e4f409b.jpg)

**Comment by InZidiuZ** [comment]
Hey,

Looks like the issue was related to incorrect number types. Anyways, I took care of what you mentioned, so will close this.

* The issue related to the "mile" values has been resolved.
* I fixed the taxi salaries. Make sure to go to life invader & reassign the job after the next patch has been pushed.
* Added checks which makes sure the values being passed are in fact numbers.

**InZidiuZ**
*OP-FW Developer*
